src/components/DayPicker.jsx

```diff
@@ -104,14 +102,63 @@ export default class DayPicker extends React.Component {
   }
 
   getMonthHeightByIndex(i) {
-    const $transitionContainer = $(ReactDOM.findDOMNode(this.refs.transitionContainer));
-    return this.getMonthHeight($transitionContainer.find('.CalendarMonth').eq(i));
+    return this.getMonthHeight(
+      ReactDOM.findDOMNode(this.refs.transitionContainer).querySelectorAll('.CalendarMonth')[i]
+    );
   }
 
-  getMonthHeight($el) {
+  getMonthHeight(el) {
+    const caption = el.querySelector('.js-CalendarMonth__caption');
+    const grid = el.querySelector('.js-CalendarMonth__grid');
+
     // Need to separate out table children for FF
-    return $el.find('.js-CalendarMonth__caption').outerHeight(true) +
-      $el.find('.js-CalendarMonth__grid').height() + 1; // for border
+    // Add an additional +1 for the border
+    return (
+      this.calculateDimension(caption, 'height', 'outer', true) +
+      this.calculateDimension(grid, 'height') + 1
+    );
+  }
+
+  applyTransformStyles(el, transform, opacity = '') {
+    const transformStyles = getTransformStyles(transform);
+    transformStyles.opacity = opacity;
+
+    Object.keys(transformStyles).forEach((styleKey) => {
+      // eslint-disable-next-line no-param-reassign
+      el.style[styleKey] = transformStyles[styleKey];
+    });
+  }
+
+  calculateDimension(el, axis, range = 'inner', withMargin = false) {
+    if (!el) {
+      return 0;
+    }
+
+    const style = window.getComputedStyle(el);
```

Comment:
Ah, I see what you mean. That's not a bad idea, but might require a bit too much coupling.